src/traces/bar/plot.js

```diff
@@ -42,9 +45,9 @@ module.exports = function plot(gd, plotinfo, cdbar) {
 
             arraysToCalcdata(d);
 
-            d3.select(this).selectAll('path')
+            d3.select(this).selectAll('g.point')
                 .data(Lib.identity)
-              .enter().append('path')
+                .enter().append('g').classed('point', true)
```

Comment:
The funny indentation this had is fairly standard with d3, to mark operations within a chain that change the selection. Not sure how consistent we've been with that but it was done that way intentionally.